__app__/crop/db.py

```diff
@@ -200,5 +200,10 @@ def session_close(session):
     Closes the current open session
     -session: an open session
     """
-    session.commit()
-    session.close()
+    try:
```

Comment:
Excellent!  We should make sure we always call this function when closing a session, rather than being tempted to just do `session.close()`.